src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -341,42 +341,64 @@ public HttpClientSettingsKey getHttpClientKey() throws SnowflakeSQLException {
       String httpsProxyHost = systemGetProperty("https.proxyHost");
       String httpsProxyPort = systemGetProperty("https.proxyPort");
       String noProxy = systemGetEnv("NO_PROXY");
+      String nonProxyHosts = systemGetProperty("http.nonProxyHosts");
       // log the JVM parameters that are being used
       if (httpUseProxy) {
```

Comment:
A small question: both http proxy and https proxy use `boolean httpUseProxy = Boolean.parseBoolean(systemGetProperty("http.useProxy"));`?

Reply:
Yes, they both use that property.